eos-tech-support/eos-convert-system-qa

```diff
@@ -154,8 +153,17 @@ if $OSTREE || $APPS; then
             --set=branches="${new_branch};"
 
         # Check for an update every time the updater runs
-        # rather than once every two weeks
-	sed -i 's/IntervalDays=14/IntervalDays=0/' /etc/eos-updater.conf
+        # rather than once every two weeks. eos-updater.conf is the old
+        # location of the config file.
+        sed -i 's/IntervalDays=14/IntervalDays=0/' /etc/eos-updater/eos-autoupdater.conf 2>/dev/null || true
+        sed -i 's/IntervalDays=14/IntervalDays=0/' /etc/eos-updater.conf 2>/dev/null || true
+
+        if [ ! -f /etc/eos-updater/eos-autoupdater.conf ] && [ ! -f /etc/eos-updater.conf ]; then
```

Comment:
This is (will be) a fairly common case: the case where a new image is installed — `/etc/eos-updater.conf` would not exist because it’s old/deprecated, and `/etc/eos-updater/*` would not exist because they need to be manually copied and edited from the defaults in `/usr/share/eos-updater/`.